packages/supermassive/.eslintrc.json

```diff
@@ -1,4 +1,23 @@
 {
-  "extends": ["../../.eslintrc.json"],
+  "extends": [
+    "../../.eslintrc.json",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "rules": {
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      { "argsIgnorePattern": "^_" }
+    ]
+  },
```

Comment:
What's different here?

Reply:
That is a merge artifact, I did eslint there first. Will fix.